admin/data/sql/migrations/20200904113433.do.create-user-input-type-table.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS mtc_results.userInputType;
+
+CREATE TABLE mtc_results.userInputType
+([id]        INT IDENTITY ( 1, 1 ) NOT NULL,
+ [createdAt] DATETIMEOFFSET(3)     NOT NULL DEFAULT GETUTCDATE(),
+ [updatedAt] DATETIMEOFFSET(3)     NOT NULL DEFAULT GETUTCDATE(),
+ [version]   ROWVERSION,
+ [name]      VARCHAR(10)           NOT NULL,
```

Comment:
please use nvarchar (as per existing tables) unless you have a specific reason for it.

Reply:
Changed to NVARCHAR for consistency with other tables.